### DISCUSSION TITLE
rate limit

### DISCUSSION THREAD

**Discussion by Teicu** [maurosoria/dirsearch#1177] [General]
Hello,  

if the application I'm testing has a rate limit and only supports 3 requests per second. How could I set dirsearch to send only 3?  

Thank you

**Comment by maurosoria** [comment]
Hello telcu,

Use the ``` --max-rate=RATE ``` flag.

Another alternative is using a single thread and a delay of ```1 second / 3 = 0. 3333~ ``` (0.34). Use ```--delay 0.34 --threads 1```.

**Comment by Teicu** [comment]
Thank you very much Mauro. I appreciate your help!
